wormhole-connect/src/views/Redeem/Confirmations.tsx

```diff
@@ -89,7 +91,21 @@ function Confirmations(props: Props) {
             {confirmations} / {chainConfig.finalityThreshold} Confirmations
           </>
         ) : route ? (
-          RoutesConfig[route].pendingMessage
+          isInvalidVaa ? (
+            <>
+              There are not enough valid signatures to repair the VAA <br />{' '}
+              Please reach out to the Wormhole support on{' '}
```

Comment:
Nit: Can we say, Please reach out to Wormhole support on

Reply:
I would advocate for saying something less technical, but this should be an extremely rare occurrence moving forward, so it's probably okay

Reply:
Sure, I've just copied from an existing message at the repo